service/src/main/resources/templates/diagnostic_report_compound_statement_template.mustache

```diff
@@ -9,12 +9,12 @@
             <center nullFlavor="NI"/>
         </effectiveTime>
         <availabilityTime value="{{availabilityTime}}"/>
-        {{#narrativeStatements}}
-            {{{.}}}
-        {{/narrativeStatements}}
         {{#specimens}}
             {{{.}}}
         {{/specimens}}
+        {{#narrativeStatements}}
```

Comment:
What's the reason for moving diagnostic report level narrative statements below specimens?

Reply:
see CompundStatement schema:

```
   <xs:complexType name="RCMR_MT030101UK04.CompoundStatement">
      <xs:sequence>
         <xs:element name="id" type="II" minOccurs="1" maxOccurs="2"/>
         <xs:element name="code" type="CD" minOccurs="1" maxOccurs="1"/>
         <xs:element name="statusCode" type="CS" minOccurs="1" maxOccurs="1"/>
         <xs:element name="effectiveTime" type="IVL_TS" minOccurs="1" maxOccurs="1"/>
         <xs:element name="availabilityTime" type="TS" minOccurs="1" maxOccurs="1"/>
         <xs:element name="priorityCode" type="CV" minOccurs="0" maxOccurs="1"/>
         <xs:element name="uncertaintyCode" type="CV" minOccurs="0" maxOccurs="1"/>
         <xs:element type="RCMR_MT030101UK04.Specimen03" minOccurs="0" maxOccurs="unbounded" name="specimen"/>
         <xs:element type="RCMR_MT030101UK04.Component02" minOccurs="1" maxOccurs="unbounded" name="component"/>
         <xs:element name="informant" type="RCMR_MT030101UK04.Informant" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="Participant" type="RCMR_MT030101UK04.Participant" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="replacementOf" type="RCMR_MT030101UK04.ReplacementOf" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="reason" type="RCMR_MT030101UK04.Reason" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="reference" type="RCMR_MT030101UK04.Reference" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="sequelTo" type="RCMR_MT030101UK04.SequelTo" minOccurs="0" maxOccurs="unbounded"/>
      </xs:sequence>
(...)
```

component element should be below specimens